sample-client-app/src/main/resources/sampleconfig.properties

```diff
@@ -1,27 +1,28 @@
-# [vipServer      ] - Singleton server's location. It could points to local Singleton server, or remote Singleton server.
-# [initializeCache] - If initializeCache is true, the translation will be loaded to application cache from Singleton service when application starts up;
-#                     else the translation will be fetched to cache when API call happens.
-# [pseudo         ] - If pseudo is true, client will return pseudo translation whether translation is ready or not.
-# [collectSource  ] - If collectSource is true, will enable the source collection functionality which means client will send the received sources to Singleton server for translation;
-#                     else there's no source collection.
-# [cleanCache     ] - If cleanCache is true, cache will be cleaned periodically.
-# [cacheExpiredTime  ] - Cache expiration time. unit: millisecond.
-# [machineTranslation] - If machineTranslation is true, will get translation by machine translation.
-######################################################################################
-vipServer = http://localhost:8090
-offlineResourcesBaseUrl = offlineBundles
-initializeCache = false
-pseudo = false
-collectSource = false
-cleanCache = false
-cacheExpiredTime = 8600000
-machineTranslation = false
-defaultLocale = fr
-sourceLocale = en
\ No newline at end of file
+# It's used for categorizing translation by product on Singleton side to maintain translation.
+productName = JavaSample
+
+# Define the translation's version used by Singleton service.
+version = 1.0.0
+
+######################################################################################
+# [vipServer      ] - Singleton server's location. It could points to local Singleton server, or remote Singleton server.
+# [initializeCache] - If initializeCache is true, the translation will be loaded to application cache from Singleton service when application starts up;
+#                     else the translation will be fetched to cache when API call happens.
+# [pseudo         ] - If pseudo is true, client will return pseudo translation whether translation is ready or not.
+# [collectSource  ] - If collectSource is true, will enable the source collection functionality which means client will send the received sources to Singleton server for translation;
+#                     else there's no source collection.
+# [cleanCache     ] - If cleanCache is true, cache will be cleaned periodically.
+# [cacheExpiredTime  ] - Cache expiration time. unit: millisecond.
+# [machineTranslation] - If machineTranslation is true, will get translation by machine translation.
+######################################################################################
+vipServer = http://localhost:8090
+offlineResourcesBaseUrl = offlineBundles
+initializeCache = false
+pseudo = false
+collectSource = false
+cleanCache = false
+cacheExpiredTime = 8600000
+machineTranslation = false
+defaultLocale = fr
+sourceLocale = en
+prodMode = true
```

Comment:
This should be false by default.

Reply:
It is false by default in the client library. This is just a sample.

Reply:
I will just manually set prodMode=true in one of the examples.